### DISCUSSION TITLE
Visual Studio Logout and Switch Accounts

### DISCUSSION THREAD

**Discussion by mikelty** [community/community#38423] [Copilot Conversations]
I intended to ask a question here cause I accidentally activated copilot on Visual Studio Community 2022 using my unpaid copilot account and was trying to switch to my paid github account. Eventually I was able to do this by using the 'search' function on Visual Studio's home page (Ctrl+Q) and typing in copilot, selecting 'sign in' and pressing 'continue' on the expiration notice window. 

I also tried disabling roaming, uninstalling / reinstalling the copilot plugin, and restarting the machine. 

Hope this helps.

**Comment by ossanna16** [comment]
Hi there @mikelty and welcome to our community! Thank you for the feedback!

**Comment by danrleypereira** [comment]
I'm sharing what worked for me
In VSCode 2022:
1. open Tools -> Options
2. search 'copilot'
3. set 'Signin with Device Flow' to 'True'
4. now open 'github.com/login/device' (remember to open it on the browser logged in the correct account) and enter the code generated in the VSCode 2022.
5. Done

![image](https://github.com/community/community/assets/12025215/5549e072-a49b-41e3-b49a-4d56b05f1a4f)
